CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to
 #### Breaking Changes
 - Return `uint32` instead of `uint64` for `pid` and `tid` builtins
   - [#3441](https://github.com/bpftrace/bpftrace/pull/3441)
+  - [Migration guide](docs/breaking.md#pid-and-tid-builtins-return-uint32)
```

Comment:
Nit: maybe this should just go as a paragraph under the breaking changes header.

Reply:
Oh sorry, guess I missed that.